lib/Data/Validate/Sanctions.pm

```diff
@@ -52,7 +52,17 @@ sub update_data {
 sub last_updated {
     my $self = shift;
     my $list = shift;
-    return $list ? $self->{_data}->{$list}->{updated} : $self->{last_time};
+    
+    if ($list) {
```

Comment:
Documentation for this method is here:

https://github.com/binary-com/perl-Data-Validate-Sanctions/blob/6d630fcdbcbd8f02e8ced7cc0d06c0f6dcfc6795/lib/Data/Validate/Sanctions.pm#L282

and needs updating to match what this is actually doing (e.g. mention that it'll load the files).

Reply:
Updated. Thanks.